packages/client/src/config.ts

```diff
@@ -653,6 +653,13 @@ export class Config {
     return key
   }
 
+  superMsg(msg: string, meta?: any) {
```

Comment:
Super Message?  :-)

Reply:
I'd like some more of these.

Reply:
Inventing new names and words is one of my favorite parts of being a developer! 😂

> I'd like some more of these.

Yes, that super simple change has an impressive effect on the possibility to orientate, right?

I was a bit cautious to not directly overload the output, also only did the first very very apparent ones, but we will likely stumble upon more "inflection points" which we want to take in here.

Reply:
("Client synced" or whatever we have here is for sure another one e.g.)